src/app/index.ts

```diff
@@ -362,6 +451,13 @@ module.exports = class extends yo {
     });
   }
 
+  _postOpenSampleHints(): void {
```

Comment:
We don't need to special case general information.

Reply:
> We don't need to special case general information.

Can we just put the special case here temporarily for sample add-ins?

Reply:
This is shipping code.  Even "temporary" things need to be ship quality.